lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -1231,6 +1246,30 @@ def cmd_favorite(*args)
             end
           end
 
+          def cmd_favorites_help
+            print_line 'Usage: favorites'
+            print_line
+            print_line 'Print the list of favorite modules (alias for `show favorites`)'
+            print @@favorites_opts.usage
+          end
+
+          #
+          # Print the list of favorite modules from the fav_modules file (alias for `show favorites`)
+          #
+          def cmd_favorites(*args)
+            if args.empty?
+              cmd_show('favorites')
+              return
+            end
+
+            # always display the help banner if the command is called with arguments
+            unless args.include?('-h')
+              print_error('Invalid option(s) provided')
+            end
+
+            cmd_favorites_help
```

Comment:
Huh, do we want to be printing out the help still in addition to this error message or should be just be returning here instead?

Reply:
Nm this actually helps otherwise the user would be confused what the correct arguments are. Marking resolved.